packages/portal-proto/src/features/genomic/FilterPanel.tsx

```diff
@@ -45,33 +46,78 @@ const GeneAndSSMFilterPanel = ({
   );
 
   return (
-    <div
-      data-testid="filters-facets"
-      className="flex flex-col gap-y-4 mt-12 max-h-screen overflow-y-auto border-t-1 border-b-1 rounded-md"
-    >
-      <GeneSetModal
-        opened={modal === Modals.LocalGeneSetModal}
-        modalTitle="Filter Mutation Frequency by Mutated Genes"
-        inputInstructions="Enter one or more gene identifiers in the field below or upload a file to filter Mutation Frequency."
-        selectSetInstructions="Select one or more sets below to filter Mutation Frequency."
-        updateFilters={updateFilters}
-        existingFiltersHook={useGenomicFacetFilter}
-      />
+    <div>
+      <Text size="lg" className="text-primary-darker font-bold my-4">
+        Filters
+      </Text>
+      <div
+        data-testid="filters-facets"
+        className="flex flex-col gap-y-4 max-h-screen overflow-y-auto border-t-1 border-b-1 rounded-md w-48 md:w-64 lg:w-80 2xl:w-96"
```

Comment:
This is going to be a merge conflict with this PR: https://github.com/NCI-GDC/gdc-frontend-framework/pull/1252 . I might need your help to figure out what these responsive changes are supposed to be and apply them to my branch.

Reply:
Sure, let me know.
Giving the facet panel a fixed width based on the screen dimension. Otherwise it kept on increasing wide.